packages/gatsby-theme-docs/gatsby-config.js

```diff
@@ -98,6 +98,14 @@ module.exports = (themeOptions = {}) => {
           path: path.resolve(`./src/releases`),
         },
       },
+      // Code examples
+      {
+        resolve: 'gatsby-source-filesystem',
+        options: {
+          name: 'codeExamples',
+          path: path.resolve(`./src/code-examples`),
+        },
+      },
```

Comment:
All gatsby plugins do not ship with the `gatsby-config` included, so this needs to be defined by the theme/website itself.